Readme.md

```diff
@@ -168,6 +171,22 @@ const logger = pino({ level: 'info' }, stream)
 logger.debug('hi')
 ```
 
+### Usage with Jest
+
+Logging with Jest is _problematic_, as the test framework requires no asynchronous operation to
```

Comment:
I wonder if Jest set any world records for issues being open in repositories that have nothing to do with the project hosted in the repository per se.

Reply:
That's my experience.